src/ohri-dashboard/ohri-dashboard.meta.tsx

```diff
@@ -0,0 +1,108 @@
+import React, { useState } from 'react';
+import { Home32, ListBulleted32, Medication32, Coronavirus32, Calendar32 } from '@carbon/icons-react';
+import { SideNavLink, SideNavMenu, SideNavMenuItem } from 'carbon-components-react';
+import { ExtensionSlot, navigate } from '@openmrs/esm-framework';
+
+export const clearDashboardSidenavRegistry = () => localStorage.removeItem('ohri-dashboard-sidenav-items');
```

Comment:
@pirupius where are you referencing this function?

Reply:
@samuelmale i need to verify if we're still using it and delete if not

Reply:
Just checked and we don't need it, i've deleted it from my working branch

Reply:
@pirupius do you want to remove it from this branch?

Reply:
@samuelmale you can approve this PR, am making another push to fix some responsive css as well as deleting that

Reply:
This has been fixed and am resolving this conversation.